yivi-docs/blog/2025-7-03-yivi-portal-announcement/index.md

```diff
@@ -0,0 +1,49 @@
+---
+slug: 2025-july-portal-announcement
+title: Yivi Portal announcement
+authors: [saravahdatipour]
+tags: [yivi, yivi-portal, trusted-verifier , mobile-app]
+---
+
+<a href="https://portal.yivi.app"> Yivi Portal </a> is Yivi's new initiative to help with a smooth launch for the Trusted/Pretty verifier. Trusted Verifier is a pivotal part of the roadmap as mentioned earlier in earlier blog posts and
+serves as part of our commitment to our users, as well as helps with transparency with our verifiers.
+In this blog post I will dive into what is the portal exactly, and what can you expect from the coming updates.
+
+
+## Yivi Portal 
+
+As part our core values, we want to make Yivi more accessible and easy to use. That also concerns developers and the business facing side of matters. 
+In the portal you can see which organizations play a part and work with Yivi, search between the credentials and personal information that are available. <br/>
+ As a verifier (referred to as relying party to adhere with the most recently used terminologies) you can register as an organization. You will then be able to set up your relying party and wait to join Yivi as a trusted verifier.
+Another feature now coppled with the portal is the new <a>attribute index</a> service you can see  
+<a href="https://attribute-index.yivi.app/">here</a>. You can also issue demo credentials as you did previously. 
+
+
+
+
+## Trusted Verifier 
+:::warning
+The Trusted Verifier will be set in motion September 1st
```

Comment:
4th of July
Maintainers from organizations can register their organization in the Yivi portal
Or request association with an existing organization

1st of September 
Mobile App Release either:
Warning users that they are sharing data with an unknown party.
Providing them feedback that they share data with a known party.
Relying parties are expected to have registered themselves and have an agreement with Yivi.